backend/src/models/File.ts

```diff
@@ -51,11 +49,14 @@ const FileSchema = new Schema<FileInterface>(
     bucket: { type: String, required: true },
     path: { type: String, required: true },
 
-    avScan: {
-      state: { type: String, enum: Object.values(ScanState), default: 'notScanned' },
-      isInfected: { type: Boolean },
-      viruses: [{ type: String }],
-    },
+    avScan: [
```

Comment:
Don't forget, this will require a migration. I don't think a migration was done when this field was originally added so this migration will have to account for when this field doesn't exist

Reply:
From looking at this diff it looks like the previous type of `avScan` was an object and now `avScan` is an array? That's why I thought this would require a migration